Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '2.7.1'
+ruby '2.7.2'
```

Comment:
This would need to be updated in the Dockerfile, too.

Reply:
You are right. Not into docker. I changed the dockerfile